Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -26,13 +26,14 @@ def summarize
         path.relative_path_from(cask.staged_path).to_s
       end
 
-      def install_phase(**options)
-        run_installer(**options)
+      def install_phase(**install_options)
+        run_installer(**install_options)
```

Comment:
I think it would be better to rename the `options` method to keep signatures consistently using `**options`.

Reply:
Convinced, thanks!